ocaml/file_formats/cms_format.ml

```diff
@@ -26,8 +26,10 @@ type cms_infos = {
   cms_comments : (string * Location.t) list;
   cms_sourcefile : string option;
   cms_builddir : string;
+  cms_loadpath : Load_path.paths;
   cms_source_digest : Digest.t option;
-  cms_uid_to_loc : Location.t Shape.Uid.Tbl.t;
+  cms_initial_env : Env.t;
```

Comment:
This field also may be storing more than necessary. If I understand correctly, it's always produced by `Typemod.initial_env`, which consults a few flags and then does its work. Instead, we could persist the flags (or get them from the build system).

Reply:
Agreed. I added a CR